building-and-research/building-and-research.md

```diff
@@ -4,3 +4,14 @@ has_children: true
 title: Building and research
 description: Building and research about machine translation
 ---
+
+Machine translation is the task of translating natural language automatically.
+The input is content in one language, and the output should be the same content in another language.
+
+The most common input and output type is text.
+There are also other [input and output types](/building-and-research/other-input-types/other-input-types.md), like speech.
+
+Most machine translation systems translate each sentence separately, without any [context](/customisation/context.md).
+Most machine translation models support only one directed language pair.
```

Comment:
In my opinion, "directed language pair" may not very clear. Maybe "Most machine translation models support language pairs in a single direction" or "Most machine translation models do not support reverse language pairs."

Reply:
Done